app/src/pages/inside/settingsPage/settingTabs/autoAnalysisTab/autoAnalysisTab.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { fetchAutoAnalysisConfigurationAction } from 'controllers/project';
+import { activeProjectSelector } from 'controllers/user';
+import { AnalysisForm } from './analysisForm/analysisForm';
+import { IndexActionsBlock } from './indexActionsBlock';
+import styles from './autoAnalysisTab.scss';
+
+const cx = classNames.bind(styles);
+
+@connect(
+  (state) => ({
+    projectId: activeProjectSelector(state),
+  }),
+  {
+    fetchAutoAnalysisConfigurationAction,
+  },
+)
+@injectIntl
+export class AutoAnalysisTab extends Component {
```

Comment:
AutoAnalysisSettings will be better name

Reply:
But we already have one settings Tab: generalTab
See the screenshot below
![image](https://user-images.githubusercontent.com/31861805/43644104-381dd45a-9736-11e8-8903-dd25d9860c3f.png)

Reply:
ok, keep current, we will resolve naming question later, when these pages will be integrated into tabs component.